Neos.Neos/Classes/FrontendRouting/NodeAddress.php

```diff
@@ -22,16 +22,6 @@
 use Neos\Flow\Annotations as Flow;
 
 /**
- * A persistent, external "address" of a node; used to link to it.
- *
- * Describes the intention of the user making the current request:
- * Show me
- *  node $nodeAggregateId
- *  in dimensions $dimensionSpacePoint
- *  in contentStreamId $contentStreamId
- *
- * It is used in Neos Routing to build a URI to a node.
- *
```

Comment:
Why did you remove those?

Reply:
because its just `@deprecated` and will be removed soon.
The statements are not true anymore: `It is used in Neos Routing to build a URI to a node.`

Reply:
Right, thanks for clarifying, I briefly confused the two `NodeAddress`es.. Good that we are getting rid of one :)